src/commands/database/mysql/dump.tsx

```diff
@@ -61,60 +50,44 @@ export class Dump extends ExecRenderBaseCommand<
     const databaseId = await withMySQLId(this.apiClient, this.flags, this.args);
     const p = makeProcessRenderer(this.flags, "Dumping a MySQL database");
 
-    const connectionDetails = await getConnectionDetailsWithPassword(
+    const name = await withConnectionDetails(
```

Comment:
Is `name` the `userName` or `databaseName`/`dbName`? 😉

Reply:
Fixed with bcef63d3954d3995131d6dc1b1cc3cbb6b66a802.